src/renderer/src/components/home/Chart/Chart.tsx

```
@@ -0,0 +1,231 @@
+import { color } from '@renderer/design/styles'
+import ReactECharts from 'echarts-for-react'
+
+const Chart = () => {
```

ESLint

[error] Missing return type on function.

(@typescript-eslint/explicit-function-return-type)